package.json

```diff
@@ -1,6 +1,7 @@
 {
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
+    "preinstall": "npx only-allow yarn",
```

Comment:
this works even if the dependencies are not yet installed. if `only-allow` is not yet installed, it will pull it as part of the preinstall step.